packages/smooth_app/lib/helpers/haptic_feedback_helper.dart

```diff
@@ -54,4 +56,21 @@ class SmoothHapticFeedback {
       return userPreferences.hapticFeedbackEnabled;
     });
   }
+
+  static String getFeedbackFormLink() {
+    final String languageCode = ProductQuery.getLanguage().code;
+    if (languageCode == 'en') {
+      return 'https://forms.gle/AuNZG6fXyAPqN5tL7';
+    } else if (languageCode == 'de') {
+      return 'https://forms.gle/vCurhD2Y3ewS1YPv5';
+    } else if (languageCode == 'es') {
+      return 'https://forms.gle/CSMmuzR8i4LJBjbM9';
+    } else if (languageCode == 'fr') {
+      return 'https://forms.gle/cTR4wqGmW7pGUiaBA';
+    } else if (languageCode == 'it') {
+      return 'https://forms.gle/9HcCLFznym1ByQgB6';
+    } else {
+      return 'https://forms.gle/AuNZG6fXyAPqN5tL7';
+    }
+  }
```

Comment:
Heyy @Omegaviv these changes look good, could we only move the `getFeedbackFormLink` method, currently it's in the haptic feedback helper, though this is only meant for vibrations not for "real" feedback.

You could create a new `UserFeedbackHelper` clas in a new` lib/helpers/user_feedback_helper.dart` file